guide/matrix-vs-discord.md

```diff
@@ -74,32 +74,34 @@ Note that Matrix does not (and cannot, due to its decentralized nature) paywall
-| Read receipts | Not supported. | **Supported.** |
+| Public read receipts | Not supported. | **Supported.** |
 | **Direct messages** | Not encrypted. | **Encrypted by default**, including VoIP. |
 | Starting a DM | Depending on privacy settings, initiating a DM requires the two users to have established "friendship" or have certain mutual "servers." Users are given the choice to accept, remove, or report a DM (since late 2021). | Initiating a DM solely requires the recipient to accept the request[^3]. Users can leave DMs anytime they wish. |
-| **Group chats** | A channel is associated with a "server." You can only join 100 "servers." | A room is standalone, but can be optionally included and associated with a Space. You can join **unlimited** amount of rooms. |
+| **Group chats** | A channel is associated with a "server." You can only join 100 "servers." | A room is standalone, but can be optionally included and associated with a Space, which is just a room linking to other rooms. You can join **unlimited** amount of rooms. |
 | VoIP in groups | Supported. | Not supported until early 2022. |
 | Organizing chats | "Servers" can be organized into folders, but each "server" can only belong to 1 folder. Channels can only be organized by the "server" owner. | Rooms can be included within an unlimited amount of Spaces. Spaces may also include other Spaces (similar to Discord's channel categories). |
-| Group chat privacy | Denying "View Channel History" permission prevents users from reading messages prior to their most recent login. | You may deny new members from reading messages prior to them being invited / joining. You may also allow or deny guest access (such as [Matrix Static](http://view.matrix.org/)) from reading messages. You may also enable encryption[^4]. |
-| Publicity | Although Discord offers its own server discovery feature, the requirements are somewhat arbitrary, so third-party services are often used. | Each homeserver has a room directory which anyone may publish to. |
+| Group chat privacy | Denying "View Channel History" permission prevents users from reading messages prior to their most recent login. However the "server" owners cannot enjoy that privacy, due to how Discord's permission system works. | You may deny new members from reading messages prior to them being invited / joining. You may also allow or deny guest access (such as [Matrix Static](http://view.matrix.org/)) from reading messages. You may also enable encryption[^4]. |
+| Server-side deletion guarantees | Messages that are removed from Discord are garbage collected from discord.com databases within less than 2 days. | As Matrix follows an event chain model, there is no true deletion. The closest thing available is redaction, which instructs the users and servers to blank the content but not the metadata of the event. Redactions are best-effort. |
```

Comment:
And do you trust them in telling the truth?

> Report-delete action actually creates a copy, meaning you have to report it while you are deleting it or before.

This is actually inconsistent, for a while Discord said that it only caches the message indefinitely without actually reporting it (you'd still have to go through the form)

I also personally think this is pointless:

1. Public rooms are free-for-all, in Discord you have modlog bots that effectively prevent your message being scrubbed server-side, and similarly in Matrix a homeserver can refuse to honour a redaction request
2. It is pointless to mention this in private encrypted rooms because a message (content, at least; metadata can be logged in some other way that does not involve the messages themselves), even if retrieved from redaction, cannot be decrypted without the help of the members

Reply:
Its useful to mention because of what it implies. Discord T&S is completely powerless to act against anything that is deleted without being reported. 

Privacy and Security are both useful things to be informed about. Also yes everyone knows that all deletions are a FFA. Also we should probably mention that its standard practice for matrix homeservers to keep all redactions "pending" in the database for a set amount of time where they will apear to clients to be executed but they wont be executed until the expiration of the hold they are on. This period is commonly is defaulted to i think 7 days.

Reply:
Fair enough, but I guess there should be a footnote explaining the FFA fact just so to be fool-proof.

Reply:
Re edit: Fair enough, I'll discount the mod bots.

> If my analysis of the matter is correct

How do you know what you see is the truth?

But this is kind of going nowhere so I guess you can keep it as-is, but please do put a footnote just to be fool-proof (well, FUD-proof, to be exact)